_data/versions.yml

```diff
@@ -1,3 +1,5 @@
+currentReleaseStream: v2.5  # vX.Y format: update during major release process
```

Comment:
Very much a nitpick, but I'd rather not have an inline comment here, opting for the comment to come first instead.

Reply:
Sure, happy to make the change. The comment was only my attempt at kicking semaphore out of its slumber. Looks like the right approach is to close/reopen the PR in github.